yt_dlp/extractor/churchofjesuschrist.py

```diff
@@ -0,0 +1,31 @@
+from .common import InfoExtractor
+
+
+class ChurchOfJesusChristIE(InfoExtractor):
+    _VALID_URL = r'https?:\/\/(?:www\.)?churchofjesuschrist\.org\/broadcasts\/watch\/(?P<id>(?:\w|-)*?)\/(?:.*)'
+    _TESTS = [{
+        'url': 'https://www.churchofjesuschrist.org/broadcasts/watch/lethbridge-alberta-member-devotional/2022/10?lang=eng',
+        'md5': '424b7a63b9c3c706d1b1309c3bda90d6',
+        'add_ie': ['BrightcoveNew'],
+        'info_dict': {
+            'id': '6313308990112',
+            'ext': 'mp4',
+            'upload_date': '20221005',
+            'uploader_id': '710857117001',
+            'timestamp': 1664994416,
+            'title': str,
+            'tags': [],
+            'live_status': str
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        brightcove_id = self._html_search_regex(r'@videoPlayer":"(?P<id>\d*?)"', webpage, 'brightcove id')
```

Comment:
Since it's all digits, I'd just do

```suggestion
        brightcove_id = self._html_search_regex(r'@videoPlayer":"(\d+)', webpage, 'brightcove id')
```

also no need for group